src/lib.js

```diff
@@ -1333,6 +1346,51 @@ export const getItems = async (
     output.armor_set_rarity = constants.RARITIES[Math.max(...armor.map((a) => helper.rarityNameToInt(a.rarity)))];
   }
 
+  // Full equipment set (4 pieces)
+  if (equipment.filter((x) => x.rarity).length === 4) {
+    let output_name;
+    let reforgeName;
+
+    // Getting equipment_name
+    equipment.forEach((equipmentPiece) => {
+      let name = equipmentPiece.display_name;
+
+      // Removing skin and stars / Whitelisting a-z and 0-9
+      name = name.replace(/[^A-Za-z0-9 -']/g, "").trim();
+
+      // Removing modifier
+      if (equipmentPiece.tag?.ExtraAttributes?.modifier != undefined) {
+        name = name.split(" ").slice(1).join(" ");
+      }
+
+      equipmentPiece.equipment_name = name;
+    });
+
+    // Getting full equipment reforge (same reforge on all pieces)
+    if (
+      equipment.filter(
+        (a) =>
+          a.tag?.ExtraAttributes?.modifier != undefined &&
+          a.tag?.ExtraAttributes?.modifier == equipment[0].tag.ExtraAttributes.modifier
+      ).length == 4
+    ) {
+      reforgeName = equipment[0].display_name.split(" ")[0];
+    }
+
+    if (equipment.filter((a) => a.equipment_name == equipment[0].equipment_name).length == 4) {
+      output_name = equipment[0].equipment_name;
+    }
+
+    // Finalizing the output
+    if (reforgeName && output_name) {
+      output_name = reforgeName + " " + output_name;
+    }
+
+    output.equipment_set = output_name;
```

Comment:
is this ever displayed?

Reply:
I don't think so, most likely will remove

Reply:
Ok